java/MODULE.bazel.lock

```diff
@@ -1,331 +1,241 @@
 {
-  "lockFileVersion": 11,
+  "lockFileVersion": 18,
```

Comment:
looks like you're using a much newer Bazel version than 7.4.0. Could you update .bazelversion too?

Reply:
Done.